package.json

```diff
@@ -7,22 +7,25 @@
     "@axa-fr/react-oidc": "^5.13.5",
     "@babel/eslint-parser": "^7.19.1",
     "@emotion/react": "latest",
-    "@emotion/styled": "v11.6.0",
-    "@mui/icons-material": "^5.6.2",
+    "@emotion/styled": "latest",
+    "@mui/icons-material": "^5.10.15",
     "@mui/material": "latest",
     "@mui/system": "^5.10.15",
     "@mui/utils": "^5.10.15",
+    "@mui/x-date-pickers": "latest",
```

Comment:
never use latest, use always a fixed version

Reply:
@Valecant to explain you why: if you use latest, build are not consistent. so i build something today, it works, i build something tomorrow, if they changed something in the code, the build fails